docs/reference/technology/connectivity/connectivity.md

```diff
@@ -13,66 +13,71 @@ For IP devices:
 - NB-IoT.
 - Bluetooth Low Energy (BLE).
 
-Non-IP devices:
+<!--- Order and headings do not reflect what're in the lists below--->
+Non-IP devices require a gateway:
 
 - LoRaWAN.
 - Cellular.
 
+<!--- What about NFC?--->
+
+<!---What if we move this to the ../technology.md file?--->
 ## Choosing your connectivity method
 
-There is a wide variety of possibilities for connected devices, and there is no globally correct connectivity method. Choose a method that fits your application needs, then choose a device that can support that method. If you're already committed to a device type, you may need to adjust your application to work within that device's connectivity constraints.
+There is a wide variety of possibilities for connected devices, and no globally correct connectivity method. Choose a method that fits your application needs, then choose a device that can support that method. If you're already committed to a device type, you may need to adjust your application to work within that device's connectivity constraints.
 
 ## Internet protocol (IP) networking
 
 Mbed OS supports various IP-based connectivity options and IP stacks.
 
 <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/ip-networking-simple.png)</span>
 
+<!---This could use a caption, IMO--->
```

Comment:
What do you think about "Connectivity stack architecture in Mbed OS"?

Reply:
Sure.